{
	"schema_version": "1.6.0",
	"id": "RHSA-2019:3421",
	"modified": "2024-11-22T12:58:46Z",
	"published": "2024-09-16T02:23:17Z",
	"related": [
		"CVE-2019-3877"
	],
	"summary": "Red Hat Security Advisory: mod_auth_mellon security, bug fix, and enhancement update",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "mod_auth_mellon",
				"purl": "pkg:rpm/redhat/mod_auth_mellon"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:0.14.0-9.el8"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2019:3421.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "mod_auth_mellon-debuginfo",
				"purl": "pkg:rpm/redhat/mod_auth_mellon-debuginfo"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:0.14.0-9.el8"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2019:3421.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "mod_auth_mellon-debugsource",
				"purl": "pkg:rpm/redhat/mod_auth_mellon-debugsource"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:0.14.0-9.el8"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2019:3421.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "mod_auth_mellon-diagnostics",
				"purl": "pkg:rpm/redhat/mod_auth_mellon-diagnostics"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:0.14.0-9.el8"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2019:3421.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:enterprise_linux:8::appstream",
				"name": "mod_auth_mellon-diagnostics-debuginfo",
				"purl": "pkg:rpm/redhat/mod_auth_mellon-diagnostics-debuginfo"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:0.14.0-9.el8"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2019:3421.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2019:3421"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com/security/updates/classification/#moderate"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.1_release_notes/"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1691125"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1691894"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1702695"
		},
		{
			"type": "ADVISORY",
			"url": "https://security.access.redhat.com/data/csaf/v2/advisories/2019/rhsa-2019_3421.json"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2019-3877"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2019-3877"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2019-3877"
		}
	]
}
